### ISSUE TITLE
Let dialer to rotate

### ISSUE THREAD

**Issue by cibersheep** [ubports/dialer-app#136]
Dialer and Clock are the two apps left to be locked in portrait mode.
Core apps at least should be able to be used in any orientation

**Comment by Flohack74** [comment]
LOL we just fixed this https://github.com/ubports/dialer-app/issues/6 ... because the dialer looks really wrong in portrait...

**Comment by mateosalta** [comment]
that isn't fixing it, this breaks landscape usage, which is critical for dash mount usage

**Comment by cibersheep** [comment]
> LOL we just fixed this #6 ... because the dialer looks really wrong in portrait...

^__^ I understand that is a quick workaround.
This is more a me-to-do reminder. I'll take a look at the UI

**Comment by dobey** [comment]
> that isn't fixing it, this breaks landscape usage, which is critical for dash mount usage

It doesn't fix the existing broken design in landscape, no, but it does prevent the existing broken design from being used in landscape. Fixing the dialer to use an adaptive layout and present an appropriate design in landscape though, could be included in future OTA.

**Comment by jonnius** [comment]
I confirm dialer in landscape mode looks good (Xerpia X RC).

**Comment by C0n57an71n** [comment]
Works well on my Nexus 5 with the latest RC for OTA15. Haven't tried if the proximity/light sensor turns off the screen since I find it impossible to call in landscape mode. Other than this, the upper notification bar might seem a bit wierd when slided down, since it covers only the right half of the screen, but I think it's a wise decission as not to clutter the screen.

**Comment by C0n57an71n** [comment]
Later edit: the screen turns off (when proximity is simulated) both in portrait and landscape mode when calling.

**Comment by TartanSpartan** [comment]
Probably too late to do more on this for OTA-15, but I like having a certain orientation (in my case, portrait) enforced which overrides the global orientation (in my case, landscape) because that works well for my preferred user interface experience. Having a way to set orientation in specific manners like these, as can be done on Android, would be a nice feature for future updates. Or perhaps it could be done within an app specifically made to handle granular orientation configurations.

**Comment by mateosalta** [comment]
I think there was a bug report for the indicator that adds the specific rotations to the rotation lock

having the app control the system rotation lock has been interesting, but I'm not sure it is the right way to proceed, rotation is basically a mess since the beginning, as coninical started with a non rotating system that apps rotated themselves, then changed to a rotating system that apps could lock rotation of the system.



- Always allow system to rotate ( unless user has set global rotation lock ) this preserves gestures working on sides of the screen user is using
- Have app only control it's own window in staged mode ( with the .desktop file orientation, or in app as needed), I think rotating to use a sideways app (which you would have to do anyway to use apps that lock rotation when you are in the opposite rotation) is better then having to rotate to use the system gestures to close a opened app or just because you switched between open apps changing the gestures between switching apps

TLDR: allow system to rotate when app is locked to an orientation
